api/src/main/java/com/github/eyefloaters/console/api/model/PartitionInfo.java

```diff
@@ -26,51 +27,53 @@ private static final class OffsetInfoOrError {
     @Schema(additionalProperties = OffsetInfoOrError.class)
     Map<String, Either<OffsetInfo, Error>> offsets;
 
-    public PartitionInfo(int partition, Node leader, List<Node> replicas, List<Node> isr) {
+    public PartitionInfo(int partition, List<PartitionReplica> replicas, Integer leaderId) {
         super();
         this.partition = partition;
-        this.leader = leader;
+        this.leaderId = leaderId;
         this.replicas = replicas;
-        this.isr = isr;
     }
 
     public static PartitionInfo fromKafkaModel(org.apache.kafka.common.TopicPartitionInfo info) {
```

Comment:
Do we need the fully qualified class name, or can we import it?

Reply:
Imported TopicPartitionInfo - I think this was left from an earlier version where there was a name clash.